omnibus/config/software/apt.rb

```diff
@@ -1,99 +0,0 @@
-# Unless explicitly stated otherwise all files in this repository are licensed
-# under the Apache License Version 2.0.
-# This product includes software developed at Datadog (https:#www.datadoghq.com/).
-# Copyright 2016-present Datadog, Inc.
-
-require './lib/cmake.rb'
-
-name 'apt'
-default_version '2.7.3'
-
-dependency 'bzip2'
-dependency 'gnutls'
-dependency 'libdb'
-dependency 'libgcrypt'
-dependency 'libiconv'
-dependency 'liblz4'
-dependency 'liblzma'
-dependency 'libxxhash'
```

Comment:
AFAICS this dependency isn't needed anymore

Reply:
Done.